### DISCUSSION TITLE
SnipeIT for MSP? (Multi Tennant, Multi Azure, SCIM)

### DISCUSSION THREAD

**Discussion by Patrick20240** [snipe/snipe-it#14139] [Q&A]
Hello, i´m really in love with the GUI and features from snipe. 
I'm thinking about how it is possible to manage different customers with the software. I would like to synchronize the users so that they are assigned to the different companies. For example, we support Office365 for several customers. Is it possible to maintain the current users from AzureAD somehow automatically via SCIM or LDAP or even some API Powershell magic? This is the only way I can assign the licenses cleanly. Basically, we only care about keeping order for internal purposes and for billing. Users from customers will not log in there or maintain their inventory themselves.

EDIT:
Is it possible to schedule automations via csv files?
I already have some basic Power automate stuff running to get infos from the O365 tennants. 

When i would have the csv files from snipe-it i could automate a import job?
Syncing users and licenses via CSV?
And i want also so sync a list from our license supplier.
I can get a csv file with all Licenses we bought for our customers.
Is it possible to sync this also?

Users, users licenses
And all global licenses from our supplier.

Thank you very much :)

**Comment by mauro-miatello** [comment]
I'm interested too.
there is LDAP integration in admin panel
If you have self-hosted, you should import csv into DB with some script, even if direct access to the DB can be dangerous
alternatively, you can import csv using API
